organisation/domain1/team1/rfc001.md

```diff
@@ -0,0 +1,19 @@
+---
+author: Alvaro Alonso
+visibility: public
+status: agreed
+---
```

Comment:
This is an example comment. Consider using https://conventionalcomments.org/ when doing so. Example:

**question**: Is this metadata about the document?

Reply:
yes, this is meta data

Reply:
test